zone/client.cpp

```diff
@@ -5566,51 +5546,61 @@ uint32 Client::GetLDoNLossesTheme(uint32 t)
 	}
 }
 
-void Client::UpdateLDoNWins(uint32 t, int32 n)
+void Client::AddLDoNLoss(uint32 theme_id)
 {
-	switch(t)
+	switch (theme_id)
 	{
-	case 1:
-		m_pp.ldon_wins_guk = n;
-		break;
-	case 2:
-		m_pp.ldon_wins_mir = n;
-		break;
-	case 3:
-		m_pp.ldon_wins_mmc = n;
-		break;
-	case 4:
-		m_pp.ldon_wins_ruj = n;
-		break;
-	case 5:
-		m_pp.ldon_wins_tak = n;
-		break;
-	default:
-		return;
+		case 1:
+			m_pp.ldon_losses_guk = 1;
```

Comment:
Should these be incrementing the losses/wins totals? Or is this for something else

Reply:
I'll change it to increment, but no matter what I sent here, as long as it's non zero, it just increments one. Incrementing will make it more clear so I'll do that.